website/docs/docs/getting-started/1-installation.md

```diff
@@ -0,0 +1,19 @@
+---
+id: installation
+title: Installing BigTest
+slug: /
+---
+
+The `bigtest` package includes everything you need to get started sogo ahead and install BigTest to your project:
+```
+$ yarn add -D bigtest
+```
+
+Once that's finished installing, you can run the following command:
+```
+$ yarn bigtest init
+```
+
+This command will help generate the `bigtest.json` file for you.
```

Comment:
Is generating the bigtest.json the only thing that happens with this command?

Reply:
Yeah. And the command will modify the bigtest.json file if it already exists. And if the bigtest.json file already exists, running bigtest init will make suggestions based on what your previous configurations are.

For example, when you run `bigtest init` for the first time (and there's no bigtest.json file), the default value for the app port will be 3000. But if you already have bigtest.json configured to run the app in 4000, running bigtest init will then go "Where would you like to run your application? 4000?"